### PR TITLE
add an attribute for aliases

### PR DESCRIPTION
**Pull request by jordane**

### CODE REVIEW

File: templates/default/apache2.conf.erb [hunk 1]
```diff
@@ -5,8 +5,8 @@
   <% if @nagios_url %>
   ServerName <%= @nagios_url %>
   <% else %>
-  ServerName      <%= node['fqdn'] %>
-  ServerAlias     nagios nagios.<%= node.chef_environment =~ /^_default$/ ? "default" : node.chef_environment%>.<%= @public_domain %>
+  ServerName      <%= node['nagios']['service_name'] %>
```
Comment (ramereth) [inline templates/default/apache2.conf.erb]: I'd rather we go with a ``['nagios']['server_name']`` attribute.

Comment (jordane) [inline templates/default/apache2.conf.erb]: Yeah, I swear I typed `server`, but apparently I didn't. Cool.

Comment (jordane) [inline templates/default/apache2.conf.erb]: Fixed.

**Comment by armiller** [comment]
:+1:

**Comment by ramereth** [comment]
@jordane You probably should add those attributes with defaults in the attributes file.

**Comment by jordane** [comment]
@ramereth I'm not sure what sane defaults would be to put in the attributes file though.

**Comment by ramereth** [comment]
@jordane I think ``nagios.example.org`` is sane defaults for server, and maybe just leave the aliases nil or empty.

**Comment by ramereth** [comment]
@jordane btw, this run fails because the aliases are nil so we probably should adjust the template to deal with that more elegantly.

**Comment by jordane** [comment]
@ramereth fixed

**Pull request closed without merging**